include/deal.II/lac/trilinos_precondition.h

```diff
@@ -153,6 +153,12 @@ namespace TrilinosWrappers
                          const dealii::parallel::distributed::Vector<double> &src) const;
 
     /**
+     * Return a shared pointer to the underlaying Trilinos Epetra_Operator.
+     * So you can use the preconditioner with unwrapped Trilinos solver.
+     */
+    std_cxx11::shared_ptr<Epetra_Operator> trilinos_operator() const;
```

Comment:
When setting preconditioner, Trilinos solver needs a pointer of Epetra_Operator.
If return a reference, then a dereference operation is needed here and
an address-of operation is needed in the AztecOO solver call.

So how about return an Epetra_Operator pointer directly.